provisioning/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
It's used for `notify: Source local settings` in couchbase and hdp roles `tasks/main.yml` under Disable THP